# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/oh-my-posh/no-notice.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Avoid checking the web for newer versions at startup as it
might be a security risk depending on your threat model.

  - NoTag <notag@maneto.fr>

--- oh-my-posh-14.12.0/src/shell/init.go.vanilla	2023-03-16 11:15:15.691485492 +0100
+++ oh-my-posh-14.12.0/src/shell/init.go	2023-03-16 11:39:05.740564239 +0100
@@ -11,7 +11,6 @@
 
 	"github.com/jandedobbeleer/oh-my-posh/src/platform"
 	"github.com/jandedobbeleer/oh-my-posh/src/template"
-	"github.com/jandedobbeleer/oh-my-posh/src/upgrade"
 )
 
 //go:embed scripts/omp.ps1
@@ -251,7 +250,9 @@
 	default:
 		return fmt.Sprintf("echo \"No initialization script available for %s\"", shell)
 	}
-	notice, hasNotice := upgrade.Notice(env)
+	//notice, hasNotice := upgrade.Notice(env)
+	notice := ""
+	hasNotice := false
 	return strings.NewReplacer(
 		"::OMP::", executable,
 		"::CONFIG::", configFile,
